packages/wrangler/src/dev/proxy.ts

```diff
@@ -636,9 +636,13 @@ function createStreamHandler(
  */
 export async function waitForPortToBeAvailable(
 	port: number,
+	host: string,
 	options: { retryPeriod: number; timeout: number; abortSignal: AbortSignal }
 ): Promise<void> {
 	return new Promise((resolve, reject) => {
+		if (host === "*") {
+			host = "0.0.0.0";
```

Comment:
I am not sure how I can trigger this code path as making `wrangler dev --remote --ip "*"` work also needed this check in `packages/wrangler/src/dev.tsx`.

Based on the following existing code, I added the explicit check for `*` here just in case:
https://github.com/cloudflare/workers-sdk/blob/749fa3c05e6b9fcaa59a72f60f7936b7beaed5ad/packages/wrangler/src/dev/proxy.ts#L169